docs/TagSeqDiagram/GetModule.puml

```diff
@@ -0,0 +1,26 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+skinparam shadowing false
+participant ":TagCommand" as TagCommand
+participant ":ModuleList" as ModuleList
+
+mainframe **sd** Get Module
+
+activate TagCommand
+
+alt Objects.isNull(taskMod)
+    TagCommand -> ModuleList: getGeneralTasks()
+    activate ModuleList
+    return generalTasks:Module
+
+else else
+    TagCommand -> ModuleList: getModule(taskModule:Module)
+    activate ModuleList
+
+    alt Objects.isNull(taskMod)
+        [<-- ModuleList: e:NoSuchModuleException
+
```

Comment:
Did you forget a return here? I think it will return the taskModule:Module if the returned is not null?